### PR TITLE
Initial react frontend

### PR DESCRIPTION
**Pull request by mateuszmarek**
install node, npm
then in `react-frontend` dir:
`npm install` or (`npm install -g yarn` and `yarn install`) yarn should be faster
`npm start`
enjoy

For additional component inspection in developer tools install: https://chrome.google.com/webstore/detail/react-developer-tools/fmkadmapgofadopljbjfkapdkoienihi?hl=en

### CODE REVIEW

File: react-frontend/public/manifest.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
```
Comment (mateuszmarek) [inline react-frontend/public/manifest.json:1]: https://developer.mozilla.org/en-US/Add-ons/WebExtensions/manifest.json

**Comment by wmilczarek** [comment]
6,5k added lines, i have no idea what it dose :+1:

**Review by wmilczarek — APPROVED** [review]
6,5k added lines, i have no idea what it dose :+1:

**Review by dkierecki — APPROVED** [review]
let's see the magic

**Review by Vortim — APPROVED** [review]

**Comment by ThePnx** [comment]
Lets see antything on fron page :p

**Review by ThePnx — APPROVED** [review]
Lets see antything on front page :p

**Pull request merged**